### PR TITLE
Add notebook config

### PR DESCRIPTION
**Pull request by jburel**
Add configuration for notebook
needed for idr0093

### CODE REVIEW

File: ansible/group_vars/omero-hosts.yml [hunk 1]
```diff
@@ -460,6 +460,15 @@ omero_web_apps_config_set:
...
         - "openmicroscopy.org/omero/bulk_annotations"
       label: "Others"
+  - menu: "notebook"
+    config:
+      default:
+        - "Study Notebook"
+      all:
+        - "Study Notebook"
+      ns:
+        - "openmicroscopy.org/mapr/analysis/notebook"
```
Comment (sbesson) [inline ansible/group_vars/omero-hosts.yml:470]: Trying to integrate this with recent efforts to expose to original analysis submitted by the authors. We have made extensive use of the `openmicroscopy.org/idr/analysis/original` namespace - see https://github.com/search?q=org%3AIDR+openmicroscopy.org%2Fidr%2Fanalysis%2Foriginal&type=code

Should this be `openmicroscopy.org/idr/analysis/notebook` instead?

Comment (jburel) [reply ansible/group_vars/omero-hosts.yml:470]: happy with the suggestion,  I will adjust the script

Comment (jburel) [reply ansible/group_vars/omero-hosts.yml:470]: script changes https://github.com/IDR/idr0093-mueller-perturbation/commit/aae814e2d3d6382312f4d4fdd1600690be74ffeb

cc @dominikl

Comment (dominikl) [reply ansible/group_vars/omero-hosts.yml:470]: 👍  Looks good to me.

**Review by sbesson — DISMISSED** [review]
Now successfully deployed on `prod93`. Proposing to get this merged and we can adjust in follow-up PRs as required?

**Comment by jburel** [comment]
Sorry i forgot to push the change to that PR
just done that now

**Review by sbesson — APPROVED** [review]
Now deployed on `prod93`

Screenshot 2021-02-08 at 10 43 14

Leaving you to confirm the annotations work as expected and we should be able to merge this

**Pull request merged**